frontend/pages/media/marketing.js

```diff
@@ -86,7 +86,7 @@ const MarketingContent = () => {
     const filteredContent = content.filter(
       (picture) =>
         selectedCategory === "All" ||
-        picture.category.split(",").includes(selectedCategory)
+        picture.category.includes(selectedCategory)
```

Comment:
Is there a reason why we're removing the split here? For marketing archives, there is usually more than one category